scripts/ci/generate-version-bump-changeset.js

```diff
@@ -18,17 +18,19 @@
 
 // This script assumes that it is being ran from the plugins workspace,
 // for example: `/workspaces/azure-devops` and would be called like this:
-// `node ../../scripts/ci/generate-version-bump-changeset.js 1.29.1`
+// `node ../../scripts/ci/generate-version-bump-changeset.js 1.29.1 minor`
 
 import fs from 'fs-extra';
 import { getPackages } from '@manypkg/get-packages';
 import { join } from 'path';
 
 async function main() {
   // Get the releaseVersion
-  const [script, releaseVersion] = process.argv.slice(1);
+  const [script, releaseVersion, versionBumpType] = process.argv.slice(1);
   if (!releaseVersion) {
```

Comment:
We should update this to test for `versionBumpType`

Reply:
@awanlin done!